components/supervisor/pkg/serverapi/publicapi.go

```diff
@@ -114,16 +114,11 @@ func NewServerApiService(ctx context.Context, cfg *ServiceConfig, tknsrv api.Tok
 		gitpodService:    gitpodService,
 		cfg:              cfg,
 		experiments:      experiments.NewClient(),
-		publicApiMetrics: grpc_prometheus.NewClientMetrics(),
+		apiMetrics:       NewClientMetrics(),
 		onUsingPublicAPI: make(chan struct{}),
 		subs:             make(map[chan *gitpod.WorkspaceInstance]struct{}),
 	}
 
-	service.publicApiMetrics.EnableClientHandlingTimeHistogram(
```

Comment:
why to remove them? current SLO relies on them?

Reply:
public api is disable for everyone so does it matter?